.github/workflows/release.yml

```
@@ -474,9 +474,20 @@ jobs:
         name: ${{ matrix.artifact-from }}-packages
         path: bazel-bin/pkg
 
+    - name: Set package architecture
+      id: pkg-arch
+      run: |
+        arch='amd64'
+        if [[ '${{ matrix.label }}' == *'arm64' ]]; then
+          arch='arm64'
+        fi
+        echo "arch=$arch"
+        echo "arch=$arch" >> $GITHUB_OUTPUT
+
     - name: Upload Packages to PULP
       env:
-        OFFICIAL_RELEASE: ${{ github.event.inputs.official }}
+        ARCHITECTURE: ${{ steps.pkg-arch.outputs.arch }}
+        OFFICIAL_RELEASE: ${{ github.event.inputs.official == true }}
```

```suggestion
        OFFICIAL_RELEASE: ${{ github.event.inputs.official }}
```

This is another issue we fixed in EE - it tries to compare `"true" == true`